app/resources/css-default.template

```diff
@@ -196,27 +196,33 @@ input {
   font-size: 13px;
   margin-bottom: .5em;
   max-width: 100%;
+}
+
+/* Show disabled input states with dotted line */
+input:not([type]):disabled {  /* not([type] means default input type */
   border: 0;
```

Comment:
for disabled input we change the underline to dotted line which is different from the input:not([type]), input[type="text"] ---> solid line and will change color when focus